nwb_conversion_tools/nwbconverter.py

```diff
@@ -110,6 +111,8 @@ def run_conversion(self, metadata: dict, nwbfile_path=None, save_to_file: bool =
 
         # If conversion_options is valid, procede to run conversions for DataInterfaces
         for interface_name, data_interface in self.data_interface_objects.items():
+            if interface_name not in conversion_options:
+                conversion_options.update({interface_name: dict()})
             data_interface.run_conversion(nwbfile, metadata, **conversion_options[interface_name])
```

Comment:
```suggestion
            data_interface.run_conversion(nwbfile, metadata, **conversion_options.get(interface_name, dict()))
```

Reply:
Good catch; as I wrote that my tired brain said there was probably a nicer way than what I was doing, but couldn't remember what it was